example/node/mint-sub-accounts-batch/name-key-year.csv

```diff
@@ -0,0 +1,7 @@
+Token_ID,To,year,,,,
+19798,0x7c6bfc5502eb719137c4c857ebdefbc49451b680,1,,,,
+19799,0xeb40c0dd1fc6c690630543ed0606e50021f78e17,1,,,,
+19800,0xe0b01f5c739d26db8d1fcf408cf444c9b1e2e52e,1,,,,
+19801,0x706d2a386a965a43d2ad0c288b67435bf4319a9b,1,,,,
+19802,0xcdccff58088eb050adc5ce9a1ac4ec037bed24cb,1,,,,
+19803,0x71772f4252b803fce311e22763c8aacc85858b0c,1,,,,
```

Comment:
This csv file format is a bit strange, ,,,, is used to indicate a line feed?

Reply:
Token_ID,To,year
19798,0x7c6bfc5502eb719137c4c857ebdefbc49451b680,1
19799,0xeb40c0dd1fc6c690630543ed0606e50021f78e17,1
19800,0xe0b01f5c739d26db8d1fcf408cf444c9b1e2e52e,1
19801,0x706d2a386a965a43d2ad0c288b67435bf4319a9b,1
19802,0xcdccff58088eb050adc5ce9a1ac4ec037bed24cb,1
19803,0x71772f4252b803fce311e22763c8aacc85858b0c,1

Reply:
They are useless, but yes, removing them would cause less confusion.